src/modules/item.js

```diff
@@ -0,0 +1,7 @@
+export default class Item {
+  constructor(index, description, completed) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index; // an intuition
+  }
+}
```

Comment:
- [ ] Please use normal functions for this project as mentioned in the project requirement, not classes, Kindly replace your classes with functions and follow the project requirement to implement the required functionalities 

![image](https://user-images.githubusercontent.com/75126481/151623767-4871c551-57f5-4617-9043-9116464e5bda.png)